eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py

```diff
@@ -107,90 +107,87 @@ def get_nextflow_process_definition(assembly_release_properties, workflow_proces
     return workflow_process_template_for_nextflow.format(**release_properties)
 
 
-def prepare_release_workflow_file_for_species(common_release_properties, taxonomy_id, assemblies, memory):
+def prepare_release_workflow_file_for_species(common_release_properties, taxonomy_id, assembly_accession, memory):
```

Comment:
```suggestion
def prepare_release_workflow_file_for_assembly(common_release_properties, taxonomy_id, assembly_accession, memory):
```

Reply:
I know I haven't replaced all the `species` with `assemblies` but I wanted to change only the minimal number of things.